app/src/views/v_services.vue

```diff
@@ -1,18 +1,89 @@
 <template>
+
   <div id="v_services">
-    {{something}} services
+
+    <div class="pageHeader">Our Services.</div>
+
+    <p>Lorem ipsum dolor sit amet, sed bibendum. Sit ut arcu, a aliquam neque, accumsan sed vivamus urna duis sit, in ac taciti curabitur amet eget bibendum. Elementum massa felis et duis</p>
+
+    <ul id="categories">
+      <li>General Contracting</li>
+      <li>Framing</li>
+      <li>Small Projects</li>
+    </ul>
+    <div class="allParagraphs">
+      <div class="paragraphContainer">
+        <div class="numbers">1.</div>
+        <h3 class="subHeader">General Contracting</h3>
+        <p>Lorem ipsum dolor sit amet, sed bibendum. Sit ut arcu, a aliquam neque, accumsan sed vivamus urna duis sit, in ac taciti curabitur amet eget bibendum. Elementum massa felis et duis</p>
+      </div>
+      <div class="paragraphContainer">
+        <div class="numbers">2.</div>
+        <h3 class="subHeader">General Contracting</h3>
+        <p>Lorem ipsum dolor sit amet, sed bibendum. Sit ut arcu, a aliquam neque, accumsan sed vivamus urna duis sit, in ac taciti curabitur amet eget bibendum. Elementum massa felis et duis</p>
+      </div>
+      <div class="paragraphContainer">
+        <div class="numbers">3.</div>
+        <h3 class="subHeader">General Contracting</h3>
+        <p>Lorem ipsum dolor sit amet, sed bibendum. Sit ut arcu, a aliquam neque, accumsan sed vivamus urna duis sit, in ac taciti curabitur amet eget bibendum. Elementum massa felis et duis</p>
+      </div>
+    </div>
   </div>
 </template>
 
 <script lang="coffee">
 module.exports =
   name: 'services',
   data: ->
-    something: 'ok'
+    something: ''
 
 </script>
 
 <style scoped lang="sass">
 #v_services
+  .allParagraphs
+    margin-left: 10%
+    margin-right: 10%
+    position: relative
+    &::before
+      content: ""
+      height: 30%
+      width: 100%
+      bottom: calc(100% + 80px)
+      background-color: whitesmoke
+      position: absolute
+      z-index: -2
+    .paragraphContainer
+      position: relative
+      margin-top: 4%
```

Comment:
use pixels for vertical margin, and only add it to one side -- so `margin-bottom: 40px` or something